docs/proposals/0017-DynamicAuth.md

```diff
@@ -0,0 +1,441 @@
+Date: 27th November 2025
+Authors: guicassolato
+Status: Provisional
+
+# Dynamic Auth
+
+This proposal extends the [Tool Authorization proposal (0008)](./0008-ToolAuthAPI.md) with additional structure designed for highly dynamic agentic applications.
+
+Unlike the static authorization model in proposal 0008—where identities and tools are explicitly enumerated in policy resources—this proposal addresses scenarios where both identities and resources are dynamic: identities may be unbounded in number, and registered without prior knowledge of the resource servers, and server resources (tools, prompts, etc.) may change frequently.
+
+A set of new fields proposed to be added the AccessPolicy API can be used in conjunction with the existing ones from proposal 0008, or independently for use cases where this level of dynamism makes explicit enumeration impractical.
+
+----
+
+🚫 **STOP – PROVISIONAL API**
+**Do NOT implement. Do NOT use in production.**
+
+This API is **provisional** and subject to change without prior notice. Vendors and integrators should not implement or rely on it, and it must not be enabled in production environments until a stable version is released.
+
+----
+
+## Non-Goals
+
+Agent and user authentication is not within the scope of this proposal.
```

Comment:
This seems incorrect since the proposal does introduce a new field `OIDC` for authN.